docs/export_kernels.rst

```diff
@@ -0,0 +1,212 @@
+Export Taichi kernels to C source
+=================================
+
+The C backend of Taichi allows you to **export Taichi kernels to C source**.
+
+The exported Taichi program consists purely of C99 compatible sources and does not require Python. Instead, you can use the exported sources in a C/C++ project, or even by calling them in Javascript via Empscripten.
+
+Each C function corresponds to a Taichi kernel.
+For example, ``Tk_init_c6_0()`` may corresponds to ``init()`` in ``mpm88.py``.
+
+Only the source is needed when copying, all the required Taichi runtimes are
+included in that single C source file.
+
+For example, this also allows commercial users to distribute their Taichi program in
+binary format by linking this file with their project.
+
+.. warning::
+
+    Currently, this feature is only officially supported on the C backend which is only released on **Linux** at this moment.
```

Comment:
```suggestion
    Currently, this feature is only officially supported on the C backend which is only released on **Linux**.
```

Reply:
We'll soon release on OS X and Windows later.